lib/opensusebasetest.pm

```diff
@@ -54,6 +55,54 @@ sub export_logs {
     save_screenshot;
 }
 
+sub export_wicked_logs {
+    my $self = shift;
+
+    # https://en.opensuse.org/openSUSE:Bugreport_wicked
+    type_string "systemctl status wickedd.service\n";
```

Comment:
better check here if a wickedd exists at all, if not, just return without action from the sub. no need to debug. Rest looks ok to me and like a good idea (although won't scale that good, e.g. considering NetworkManager or so)